src/redux/selectors/movies.ts

```diff
@@ -0,0 +1,3 @@
+import { RootState } from 'redux/store';
+
+export const moviesSelector = (state: RootState) => state.movies.movies;
```

Comment:
Not really agree with this name: your function name should **always** start with verb. I advise you to start naming selectors with `select` verb, in this case, your name will look like: `selectMovies`. Please, update it.